DomainContent/LoadTest/VoteApp/voteApp.js

```diff
@@ -0,0 +1,411 @@
+"use strict";
+/* eslint-disable indent */
+
+//
+//  Example Vue App
+//
+//  Created by Milad Nazeri on 2018-10-11
+//  Modified from AppUi.js by Howard Stearns on 2 Nov 2016
+//  Copyright 2016 High Fidelity, Inc.
+//
+//  Distributed under the Apache License, Version 2.0.
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+//
+/* globals Tablet, Script, HMD, Controller, Menu */
+
+(function () { // BEGIN LOCAL_SCOPE
+    // Dependencies
+    // /////////////////////////////////////////////////////////////////////////
+    var AppUi = Script.require('./AppUi.js');
+    var Helper = Script.require('https://raw.githubusercontent.com/highfidelity/hifi-content/master/Utilities/Helper.js');
+
+    // Consts
+    // /////////////////////////////////////////////////////////////////////////
+    var URL = Script.resolvePath("./html/tablet.html"),
```

Comment:
coding standards: please declare/define variables as close to when they are first used as is reasonable